wagtail/admin/compare.py

```diff
@@ -172,19 +188,19 @@ def htmldiff(self):
             classes = ['comparison__child-object']
             if comparison.is_new():
                 classes.append('addition')
-                block_rendered = comparison.block.render_basic(comparison.val_b)
+                block_rendered = comparison.htmlvalue(comparison.val_b)
             elif comparison.is_deleted():
                 classes.append('deletion')
-                block_rendered = comparison.block.render_basic(comparison.val_a)
+                block_rendered = comparison.htmlvalue(comparison.val_a)
             elif comparison.has_changed():
                 block_rendered = comparison.htmldiff()
             else:
-                block_rendered = comparison.block.render_basic(comparison.val_a)
+                block_rendered = comparison.htmlvalue(comparison.val_a)
 
             classes = ' '.join(classes)
             comparisons_html.append(f'<div class="{classes}">{block_rendered}</div>')
```

Comment:
Travis [build is failing](https://travis-ci.org/wagtail/wagtail/builds/476294238?utm_source=github_status&utm_medium=notification) for Python < 3.6 because [f-strings aren't available](https://www.python.org/dev/peps/pep-0498/) in those older versions.

Reply:
Oops, I got into this habbit now.. Thanks!